doc/tutorials/dune-package-management/pinning.md

```diff
@@ -41,10 +44,11 @@ Solution for dune.lock:
 Unlike previously, the version of the `fmt` library that is picked is `dev`, to
 signify a development version.
 
-The next build will check out the sources from that repository instead of
-downloading the release tarball:
+The next time the project is built, the `fmt` package will be built from the
+source in the specified git repository rather than from the source tarball
+released in the opam repository.
```

Comment:
```suggestion
source in the specified Git repository rather than from the source tarball
released in the `opam-repository`.
```

Should this be `opam-repository`? Is that different than when it's written "opam repository"? Are those two different things?